changelog/unreleased/ocs-share-api-userid.md

```diff
@@ -0,0 +1,6 @@
+Change: replace the user uuid with the username in ocs share responses
+
+The ocs api should not send the users uuid. Replaced the uuid with the username.
+
+https://github.com/owncloud/ocis/issues/990
+https://github.com/cs3org/reva/pull/1370
```

Comment:
You need to update the PR link to `1375` now ;-)

Reply:
Ah damn it